### DISCUSSION TITLE
Lesson 6: Brownie not detecting my local ganache

### DISCUSSION THREAD

**Discussion by guillermo-palou** [smartcontractkit/full-blockchain-solidity-course-py#346] [Q&A] [answered]
Hello! 

I am facing an issue that Brownie is not detecting my local Ganache. 
Here is the error:

PS C:\Users\Guillermo Palou\demos\brownie_fund_me> brownie run scripts/deploy.py --network ganache-local
INFO: Could not find files for the given pattern(s).
Brownie v1.17.0 - Python development framework for Ethereum

BrownieFundMeProject is the active project.

Running '\Users\Guillermo Palou\demos\brownie_fund_me\scripts\deploy.py::main'...
  File "C:\Users\Guillermo, line line, in in
    return_value, frame = run(
  File "C:\Users\Guillermo, line line, in in
    return_value = f_locals[method_name](*args, **kwargs)
  File "\Users\Guillermo, line line, in in
    deploy_fund_me()
  File "\Users\Guillermo, line line, in in
    price_feed_address = config["networks"][network.show_active()][
KeyError: 'ganache-local'

I already did the step of adding the ganache-local network . I get this when I run brownie networks list

PS C:\Users\Guillermo Palou\demos\brownie_fund_me> brownie networks list
INFO: Could not find files for the given pattern(s).
Brownie v1.17.0 - Python development framework for Ethereum

The following networks are declared:

Ethereum
  ├─Mainnet (Infura): mainnet
  ├─Ropsten (Infura): ropsten
  ├─Rinkeby (Infura): rinkeby
  ├─Goerli (Infura): goerli
  ├─Kovan (Infura): kovan
  └─ganache-local: ganache-local

I was able to fix the error above by adding the ganache-local into my brownie-config.yaml file. However whenever I run I still get an error.

PS C:\Users\Guillermo Palou\demos\brownie_fund_me> brownie run scripts/deploy.py --network ganache-local
INFO: Could not find files for the given pattern(s).
Brownie v1.17.0 - Python development framework for Ethereum

BrownieFundMeProject is the active project.

Running '\Users\Guillermo Palou\demos\brownie_fund_me\scripts\deploy.py::main'...
  File "C:\Users\Guillermo, line line, in in
    return_value, frame = run(
  File "C:\Users\Guillermo, line line, in in
    return_value = f_locals[method_name](*args, **kwargs)
  File "\Users\Guillermo, line line, in in
    deploy_fund_me()
  File "\Users\Guillermo, line line, in in
    price_feed_address = config["networks"][network.show_active()][
KeyError: 'eth_usd_price_feed'

It seems that my problem is when trying to retrieve the eth USD price.

![image](https://user-images.githubusercontent.com/93664299/141427716-e212a12b-c9fb-4829-9425-4dac0bb46587.png)
![image](https://user-images.githubusercontent.com/93664299/141427773-14481c36-cf6a-45cc-925c-cab13ecce84c.png)

**Comment by cromewar** [comment, marked as answer]
Hello @guillermo-palou as you said you should have added ganache-local to your brownie-config and you already did it, so everything ok on that part. Regarding to `eth_usd_price_feed` is because you only set that for rinkeby:

![image](https://user-images.githubusercontent.com/20199180/141471831-720058dc-bed3-4ca2-b720-ef19e2f6193c.png)

add the same line on ganache-local or it won't work.

Note: remember you have to deploy mocks to work with `eth_usd_price_feed` on a local ganache.
